src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -0,0 +1,90 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+@Entity
+public class Recipe extends AbstractEntity {
+
+	private HashMap<String, String> ingredient;
+
+	private String name;
+	private int servings;
+	private int cookTime;
+	private int prepTime;
+	private String images;
+
+	//@ManyToMany
+	private ArrayList<Recipe> recipes;
```

Comment:
Would we keep a list of recipes in the recipe class? This logic might make more sense in the User class. But, we could switch it around to include a list of Users who have this recipe/who have added this recipe. Might also be something we develop out a bit later.